doc/source/container_builder.rst

```diff
@@ -0,0 +1,282 @@
+Building in a Self-Contained Environment
+========================================
+
+.. topic:: Abstract
+
+   Users building images with KIWI face problems if they want
+   to build as non root user or can't install KIWI on their
+   system due to inappropriate build host requirements or
+   project policies.
```

Comment:
Well the 'or' is correct, there are users who have issues to run a program as root, others have issues installing KIWI on their system because it does not exist for this distribution, and then again there are users who have issues with both. I think saying that everybody is facing all of these issues wouldn't be correct ?

If only one of the 'or' list applies it makes sense to read this article, that's what should be expressed